### PR TITLE
Updated Docker Machine release notes

### PR DESCRIPTION
**Pull request by shin-**
### Proposed changes

Updated Docker Machine release notes

### Unreleased project version (optional)

0.13.0

### Related issues (optional)

#4871

### CODE REVIEW

**Comment by mstanleyjones** [comment]
Please rebase. Sorry the upstream branch got out of date.

File: machine/reference/mount.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+```
+
+
+Now you can use the directory on the machine, for mounting into containers.
+Any changes done in the local directory, will be reflected in the machine too.
+
+```none
+$ docker run -v /home/docker/foo:/tmp/foo busybox ls /tmp/foo
+bar
+$ docker touch foo/baz
```
Comment (gbarr01) [inline machine/reference/mount.md]: `docker touch foo/baz` should be `touch foo/baz`

```
docker: 'touch' is not a docker command.
See 'docker --help'
```

File: machine/reference/mount.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+$ docker-machine ssh dev ls foo
+bar
+```
+
+
+Now you can use the directory on the machine, for mounting into containers.
+Any changes done in the local directory, will be reflected in the machine too.
+
+```none
+$ docker run -v /home/docker/foo:/tmp/foo busybox ls /tmp/foo
```
Comment (gbarr01) [inline machine/reference/mount.md]: where are we when running this command? Perhaps prepend this block with: 
`docker-machine ssh dev`

File: machine/reference/mount.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+bar
+$ docker touch foo/baz
+$ docker run -v /home/docker/foo:/tmp/foo busybox ls /tmp/foo
+bar
+baz
+```
+
+The files are actually being transferred using `sftp` (over an ssh connection),
+so this program ("sftp") needs to be present on the machine - but it usually is.
```
Comment (gbarr01) [inline machine/reference/mount.md]: It wasn't on my Dell XPS 13 with Ubuntu Xenial ...

File: release-notes/docker-machine.md [hunk 4]
```diff
@@ -5,6 +5,93 @@ keywords: release notes, machine
...
 ---
 
+## 0.13.0 (2017-10-12)
+
+### General
+
+- Added new `docker-machine mount` command for mounting machine directories over SSHFS
+- Improved some logging messages
+- Fixed a bug with the `scp` command when using an identity file.
+-Fixed a parsing error that caused the boot2docker ISO cache to malfunction, forcing a new download everytime.
```
Comment (gbarr01) [inline release-notes/docker-machine.md]: Indent Fixed a parsing error ...?

**Comment by shin-** [comment]
Reopened cleanly here: https://github.com/docker/docker.github.io/pull/5012

@gbarr01 Sorry, I hadn't done the rebase yet, a lot of the diffs you see are artifacts from other pull requests and don't reflect my changes.

**Pull request closed without merging**